packages/expo/src/MailComposer/ExponentMailComposer.web.ts

```diff
@@ -0,0 +1,33 @@
+import array from 'cast-array';
```

Comment:
Instead of pulling in another dependency, just write out the function. Not sure we even need this casting anyway, but if so, it's like 2-3 lines w/Array.isArray.